src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -424,101 +419,97 @@ limitations under the License.
-      <description>
-        Command description for SetpointRaiseLower
-      </description>
-      <arg name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
-      <arg name="Amount" type="int8s"/>
+      <description>Upon receipt, the attributes for the indicated setpoint(s) SHALL have the amount specified in the Amount field added to them.</description>
+      <arg id="0" name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
+      <arg id="1" name="Amount" type="int8s"/>
     </command>
+
     <command source="client" code="0x01" name="SetWeeklySchedule" optional="true">
-      <description>
-        Command description for SetWeeklySchedule
-      </description>
+      <description>Upon receipt, the weekly schedule for updating setpoints SHALL be stored in the</description>
       <access op="invoke" privilege="manage"/>
-      <arg name="NumberOfTransitionsForSequence" type="int8u"/>
-      <arg name="DayOfWeekForSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
-      <arg name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
-      <arg name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
+      <arg id="0" name="NumberOfTransitionsForSequence" type="int8u"/>
+      <arg id="1" name="DayOfWeekForSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg id="2" name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <arg id="3" name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
     </command>
+
     <command source="client" code="0x02" name="GetWeeklySchedule" response="GetWeeklyScheduleResponse" optional="true">
-      <description>
-        Command description for GetWeeklySchedule
-      </description>
-      <arg name="DaysToReturn" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
-      <arg name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <description>Upon receipt, the unit SHOULD send in return the Get</description>
+      <arg id="0" name="DaysToReturn" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg id="1" name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
     </command>
+
     <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true">
-      <description>This command is used to clear the weekly schedule. The ClearWeeklySchedule command has no payload.</description>
+      <description>This command is used to clear the weekly schedule.</description>
       <access op="invoke" privilege="manage"/>
-    </command>  
+    </command>
+
     <command source="client" code="0x05" name="SetActiveScheduleRequest" optional="true">
-      <description>This command is used to set the active schedule.</description>
-      <arg name="ScheduleHandle" type="octet_string" length="16"/>
+      <description>Upon receipt, if the </description>
```

Comment:
Please make sure to fix this.  Followup is OK.  But these descriptions are in fact used, and this new description is nonsense.